src/util.h

```diff
@@ -346,17 +346,19 @@ inline v8::Local<v8::String> OneByteString(v8::Isolate* isolate,
                                            std::string_view str);
 
 // Used to be a macro, hence the uppercase name.
-template <int N>
-inline v8::Local<v8::String> FIXED_ONE_BYTE_STRING(
-    v8::Isolate* isolate,
-    const char(&data)[N]) {
+template <std::size_t N>
+  requires(N > 0)
```

Comment:
nit: can we make this a `concept` instead to avoid the duplication

Reply:
Do you have an example of how this might work? The only C++20 solution I can think of is this, but I assume it's not what you had in mind:

```cpp
template <std::size_t N>
concept IsPositiveSize = (N > 0);

template <std::size_t N>
requires IsPositive<N>
inline v8::Local<v8::String> FIXED_ONE_BYTE_STRING(...) {
  ...
}
```

Reply:
@tniessen Maybe @jasnell meant that instead of having both `const char (&data)[N]` and `const std::array<char, N>& arr`, we could have just one template function that supports both.

Trivially... something like this would do...


```cpp
template <std::size_t N, typename T>
  requires(N > 0)
inline v8::Local<v8::String> FIXED_ONE_BYTE_STRING(
    v8::Isolate* isolate, const T& arr) {
  CHECK_EQ(arr[N - 1], '\0');
  return OneByteString(isolate, arr.data(), N - 1);
}
```



And then you could turn `T` into a concept which must represent something that works... 

This might be overkill?